### ISSUE TITLE
Game crashes when I open

### ISSUE THREAD

**Issue by Chezae** [Escartem/AmongUsCheatClient#24]
Whenever I press the delete key the console begins to load and then my among us crashes

**Comment by Escartem** [comment]
Version of the game, version of the client ?

**Comment by BurnhamR** [comment]
> 
> 
> Version of the game, version of the client ?

v2020.10.22s and AUCC 0.0.3-beta.


I am having this issue as well with the newest update of Among Us

**Comment by Escartem** [comment]
the latest versions of the game are not supported yet.
You need 2020.9.22s or older to use the client
Sorry for the inconvenience

**Comment by BurnhamR** [comment]
> 
> 
> the latest versions of the game are not supported yet.
> You need 2020.9.22s or older to use the client
> Sorry for the inconvenience

Yup, I know, that's why I didn't open an issue. I just saw this issue was opened and responded with the version that it happens with.
